bundles/framework/printout/instance.js

```diff
@@ -180,6 +191,37 @@ Oskari.clazz.define('Oskari.mapframework.bundle.printout.PrintoutBundleInstance'
         getService: function () {
             return this.printService;
         },
+        showPanel: function () {
+            const controller = this.handler?.getController();
+            controller.showPanel(
```

Comment:
Optional chaining in `this.handler?.getController()` doesn't really make sense here since controller.showPanel() is called on the next line.